docs/pages/api-docs/click-away-listener.js

```diff
@@ -1,23 +0,0 @@
-import * as React from 'react';
-import ApiPage from 'docs/src/modules/components/ApiPage';
-import mapApiPageTranslations from 'docs/src/modules/utils/mapApiPageTranslations';
-import jsonPageContent from './click-away-listener.json';
-
-export default function Page(props) {
```

Comment:
We forgot to add a redirection for this file, please let's always redirect older URLs. It was raised in https://app.ahrefs.com/site-audit/3524616/10/data-explorer?columns=pageRating%2Curl%2Ctraffic%2ChttpCode%2Cdepth%2Ccompliant%2CincomingAllLinks%2Corigin&filterCollapsed=true&filterId=c8b4cd2c48fbde5a00dcef1da5e81f8b&issueId=c64dae3f-d0f4-11e7-8ed1-001e67ed4656&sorting=-pageRating. I'm taking care of it.

Reply:
Thanks for taking care of it!